### PR TITLE
Update for 1.5.0. Update port forwarding feature and reorganize conte…

### PR DESCRIPTION
**Pull request by btat**
…nt for new Preferences menu

Signed-off-by: Billy Tat <btat@suse.com>

### CODE REVIEW

File: docs/getting-started/features.md [hunk 1]
```diff
@@ -2,206 +2,230 @@
...
+Set the port Kubernetes is exposed on. Use this setting to avoid port collisions if multiple instances of K3s are running.
+
+### Enable Traefik
+
+This option allows you to enable or disable Traefik. By disabling Traefik, you can free up port 80 and 443 for alternate ingress configuration. By default, Traefik is enabled.
 
-1. Click **Reset**.
-1. On the confirmation window, click **OK** to proceed. Kubernetes stops and Rancher Desktop closes.
-1. Start Rancher Desktop again.
+To enable/disable Traefik, just check/uncheck the `Enable Traefik` checkbox. The app needs to be manually restarted when you enable/disable Traefik. This behavior will be changed in an upcoming release, where a prompt will let you restart the app. Disabling Traefik will not delete existing resources.
```
Comment (rak-phillip) [inline docs/getting-started/features.md:231]: We could probably remove a majority of this paragraph

```suggestion
Disabling Traefik will not delete existing resources.
```

File: docs/getting-started/features.md [hunk 2]
```diff
@@ -2,206 +2,230 @@
...
-    > Note: By default, images are pulled from [Docker Hub] in which use the `repo/image[:tag]` format. To pull from other registries, include the hostname `registry.example.com/repo/image[:tag]`.
-1. Click **Pull**.
+### Enable Kubernetes
 
-[Docker Hub]:
-https://hub.docker.com/
+This option allows you to enable or disable Kubernetes. By disabling Kubernetes, you can run just `containerd` or `dockerd` by itself for reduced resource consumption. By default, Kubernetes is enabled.
 
-#### Building Images
+To enable/disable Kubernetes, just check/uncheck the `Enable Kubernetes` checkbox. The app would be restarted when you enable/disable Kubernetes. Disabling Kubernetes will not delete existing resources and they will be available again when you enable Kubernetes back.
```
Comment (rak-phillip) [inline docs/getting-started/features.md:200]:
```suggestion
To enable/disable Kubernetes, just check/uncheck the `Enable Kubernetes` checkbox. The app will be restarted when you enable/disable Kubernetes. Disabling Kubernetes will not delete existing resources and they will be available again when you enable Kubernetes again.
```

File: docs/getting-started/features.md [hunk 3]
```diff
@@ -2,206 +2,230 @@
...
+Set the [container runtime] for Rancher Desktop. Users have the option of [containerd] which provides namespaces for containers and the use of nerdctl or [dockerd (moby)] which enables the Docker API and the use of the Docker CLI. Only one container runtime will function at a time.
 
-1. From the image list, find the image you want to scan.
-1. Click **⋮ > Scan**.
-1. Review the results then click **Close Output to Continue**.
+When switching to a different container runtime:
 
-[Trivy]:
-https://github.com/aquasecurity/trivy
+- A restart of Kubernetes is required.
```
Comment (rak-phillip) [inline docs/getting-started/features.md:182]: I'm not sure if we really need to warn of Kubernetes restarts like this anymore. I guess it's technically true, it's just that the restart is automatically triggered on apply.

**Review by rak-phillip — CHANGES_REQUESTED** [review]
I think this is looking good. We might want to remove some outdated info along with some grammatical improvements that I suggested.

File: docs/getting-started/features.md [hunk 4]
```diff
@@ -2,206 +2,230 @@
...
 
-When upgrading:
+To scan an image:
 
-- A check is performed to see if the target Kubernetes version is available locally. If not, it downloads the files.
-- Workloads are retained.
-- Images are retained.
+1. From the image list, find the image you want to scan.
+1. Click **⋮ > Scan**.
+1. Review the results then click **Close Output to Continue**.
```
[the remark this answers is not in the document]
Comment (btat) [reply docs/getting-started/features.md]: Updated to reflect the current process.

File: docs/getting-started/features.md [hunk 5]
```diff
@@ -2,206 +2,230 @@
...
+Use this option to pull images from a registry to your virtual machine.
 
-### Enable Kubernetes
+To pull an image:
 
-This option allows you to enable or disable Kubernetes. By disabling Kubernetes, you can run just `containerd` or `dockerd` by itself for reduced resource consumption. By default, Kubernetes is enabled.
+1. Click the **+** button in the top-right corner.
+1. Click on the **Pull** tab.
+1. Enter the name of the image to pull.
+    > Note: By default, images are pulled from [Docker Hub] in which use the `repo/image[:tag]` format. To pull from other registries, include the hostname `registry.example.com/repo/image[:tag]`.
```
[the remark this answers is not in the document]
Comment (btat) [reply docs/getting-started/features.md]: Feedback applied.

File: docs/getting-started/features.md [hunk 6]
```diff
@@ -2,206 +2,230 @@
...
+- Workloads are removed.
+- Images are retained.
 
-### Factory Reset
+To switch versions:
 
-Remove the cluster and all other Rancher Desktop settings. The initial setup procedure must be done again.
+1. Click the **Kubernetes version** drop-down menu.
+1. Select the version you want to change to.
+1. On the confirmation window, click **OK** to proceed.
```
[the remark this answers is not in the document]
Comment (btat) [reply docs/getting-started/features.md]: Removed as suggested.

File: docs/getting-started/features.md (hunk 1 above)
Comment (btat) [reply docs/getting-started/features.md]: Feedback applied.

File: docs/getting-started/features.md (hunk 3 above)
Comment (btat) [reply docs/getting-started/features.md]: Feedback applied.

File: docs/getting-started/features.md [hunk 7]
```diff
@@ -2,206 +2,230 @@
...
 
-![](/img/reset-kubernetes.png)
+In order to use the utilities, `~/.rd/bin` must be in the `PATH` variable in your shell.
 
-## WSL Integration (Windows)
+There are two options for doing this:
 
-The **WSL Integration** tab gives the option to make the Rancher Desktop Kubernetes configuration accessible to any Linux distributions configured for WSL. Once enabled, you can use communicate with the Rancher Desktop Kubernetes cluster using tools like `kubectl` from within the WSL distribution.
+- **Automatic**: `PATH` management will add `~/.rd/bin` to your `PATH` by modifying your shell .rc files for you.
+- **Manual**: `PATH` management does not change anything - in this mode you must add `~/.rd/bin` to your `PATH` yourself.
```
[the remark this answers is not in the document]
Comment (btat) [reply docs/getting-started/features.md]: Feedback applied.

File: docs/getting-started/features.md (hunk 2 above)
Comment (btat) [reply docs/getting-started/features.md]: Feedback applied.

**Review by rak-phillip — APPROVED** [review]
LGTM

**Pull request merged**